### ISSUE TITLE
The introduction of the quantum part reduces the precision

### ISSUE THREAD

**Issue by Eknaug** [hchsu/QML_CIFAR10#2]
The classical convolution easily achieves an accuracy of 0.9 or more when I try to run the code. However, once it is migrated to the quantum part, the success rate of code detection quickly drops below 0.6. And the success rate of detection barely improves for the rest of the study. Is this a flaw in the quantum part itself? Or am I calling it incorrectly?

The results of the run are as follows：
Files already downloaded and verified
Files already downloaded and verified
False
.....train classical .......
train Loss: 0.6725 Acc: 0.6015
train Loss: 0.6089 Acc: 0.6663
train Loss: 0.5539 Acc: 0.7146
train Loss: 0.5211 Acc: 0.7373
train Loss: 0.4715 Acc: 0.7723
train Loss: 0.4492 Acc: 0.7859
train Loss: 0.3998 Acc: 0.8174
train Loss: 0.3691 Acc: 0.8404
train Loss: 0.3255 Acc: 0.8611
train Loss: 0.2942 Acc: 0.8747

Test set: Average loss: 0.5296, Accuracy: 1512/2000 (76%)

Files already downloaded and verified
Files already downloaded and verified
training and testing starts...
Epoch 0/4
----------
train Loss: 0.6882 Acc: 0.5527
val Loss: 0.6800 Acc: 0.5730

Epoch 1/4
----------
train Loss: 0.6775 Acc: 0.5780
val Loss: 0.6751 Acc: 0.5730

Epoch 2/4
----------
train Loss: 0.6742 Acc: 0.5787
val Loss: 0.6745 Acc: 0.5770

Epoch 3/4
----------
train Loss: 0.6737 Acc: 0.5745
val Loss: 0.6729 Acc: 0.5780

Epoch 4/4
----------
train Loss: 0.6728 Acc: 0.5789
